### ISSUE TITLE
Set up proper CD pipeline

### ISSUE THREAD

**Issue by HeinousTugboat** [danielyxie/bitburner#3295]
Per hydro, adding an issue for this: we should add a proper pipeline so we can have both a beta and prod release, allowing us to release features to a limited set of users before more broadly pushing them out to the full userbase. Additionally, we should use envvars to track the deployed sha so we can avoid the chicken-and-egg issue of not knowing the _current_ sha of the deployed code.

**Comment by marcjmiller** [comment]
This is up my alley, what does the current process look like? I don't see a `.gitlab-ci.yml` or anything, so am I to assume it's a 100% manual job right now? I'm not familiar with building electron apps, but walk me through the steps and I'll at least get it started.

**Comment by phyzical** [comment]
we are working on this, if you checkout the .github/workflows there is some. the main issue we currently have is lack of full access to handle secrets ect. soon we will be moving to an org to gain this control.

TLDR; i wouldn't worry about it at this time
